core/src/main/java/feast/core/dao/JobRepository.java

```diff
@@ -41,5 +41,5 @@
   List<Job> findByFeatureSetJobStatusesIn(List<FeatureSetJobStatus> featureSetsJobStatuses);
 
   // find jobs by feast store name
-  List<Job> findByStoresName(String storeName);
+  List<Job> findByJobStoresIdStoreName(String storeName);
```

Comment:
I think the comment is out of date.

Reply:
why? it still looking by store name, but it could be one of stores